.github/workflows/js-checks.yml

```diff
@@ -15,23 +15,11 @@ concurrency:
 
 jobs:
   js-checks:
+    uses: hemilabs/actions/.github/workflows/js-checks.yml@main
```

Comment:
The ones in bloq/actions are improved versions over these ones. And use a tag or hash as reference, not a branch name.

Reply:
`bloq/actions` is almost exactly the same as `hemilabs/actions`.  
There are no tags on `hemilabs/actions`, so using a hash instead. Updated here [5995ab4](https://github.com/hemilabs/token-list/pull/30/commits/5995ab45075f3e5ecfaac73acac789983f61817c)